helm/idol-eductionserver/resources/eductionserver.cfg

```diff
@@ -0,0 +1,37 @@
+// patching config file
+// idol_ssl.cfg is written by run_idol.sh with SSL settings if required by IDOL_SSL env variable
+// original.eductionserver.cfg is the config file as contained in the standard component zip
+
+// modified config sections
+
+< "/eductionserver/cfg/idol_ssl.cfg" [SSLSettings]
+
+[License] < "/eductionserver/cfg/original.eductionserver.cfg" [License]
+LicenseServerHost={{ .Values.licenseServerHostname }}
+LicenseServerACIPort={{ .Values.licenseServerPort | int }}
+
+[Service] < "/eductionserver/cfg/original.eductionserver.cfg" [Service]
+Access-Control-Allow-Origin=*
+ServicePort={{ .Values.servicePort | int }}
+
+[Server] < "/eductionserver/cfg/original.eductionserver.cfg" [Server]
+Port={{ .Values.aciPort | int }}
+< "/eductionserver/cfg/idol_ssl.cfg" [EnableSSL] SSLConfig
+
+[AuthorizationRoles] < "/eductionserver/cfg/original.eductionserver.cfg" [AuthorizationRoles]
+
+[AdminRole] < "/eductionserver/cfg/original.eductionserver.cfg" [AdminRole]
+Clients=*
+
+[QueryRole] < "/eductionserver/cfg/original.eductionserver.cfg" [QueryRole]
+Clients=*
+
+[Logging] < "/eductionserver/cfg/original.eductionserver.cfg" [Logging]
+LogDirectory=/idol-logs
+LogArchiveDirectory=/idol-logs/archive
+LogEcho=TRUE
+
+[Eduction]
+ResourceFiles=/eductionserver/grammars/internet.ejr

```

Comment:
I can't see this file in the `eductionserver:24.1` image;
```
[idoluser@8a29fe76ce23 eductionserver]$ ls /eductionserver/grammars
ls: cannot access '/eductionserver/grammars': No such file or directory
```
Are we supposed to be getting the grammar into the container, somehow?

Reply:
Yeah this was just for my local testing, I've got a config map locally to get this in. Will this section once Hasan is done with the pii config and James has updated the image with the grammars/scripts.